app/models/environment.rb

```diff
@@ -49,14 +49,36 @@ def locked?
     active_lock.present?
   end
 
+  # Returns true if the provided name, matches the canonical environment name,
+  # or one of it's aliases.
+  def match_name?(name)
+    return true if self.name == name
+    return true if config.aliases.include?(name)
+    false
```

Comment:
Only reason is to make the method consistent with the type it returns (true/false). If I leave it out, it'd return nil. In Ruby, that's technically fine, because both `nil` and `false` are considered "falsey".

Reply:
I guess what I'm trying to say is, explicit `return false` instead of implicit `false` if the block already has other return statement conditions.